invenio_audit_logs/config.py

```diff
@@ -62,3 +55,11 @@
 
 AUDIT_LOGS_ENABLED = False
 """Feature flag. Disabled by default due to experimental nature of the APIs. Feature is not fully stable."""
+
+AUDIT_LOGS_METADATA_FIELDS = {
```

Comment:
Missing some docstring on what is the format expected here and how this config is used...